### PR TITLE
Initial draft documents

### PR DESCRIPTION
**Pull request by ElijahCaine**
refs #3

### CODE REVIEW

File: docs/source/draft/design.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+#. A job scheduler provokes IAM to collect data from all nodes every 30
+   minutes.
+
+    This can be provoked by Chef, Cron, whichever.
+
+#. A list of all resources is collected with their corresponding measurements
+   (CPU, Memory, Bandwidth, etc).
+
+#. For each measurement corresponding to each resource, that resource's plugin
```
Comment (Kennric) [inline docs/source/draft/design.rst]: The measurement plugin, I think - resources don't have plugins, only measurements

File: docs/source/draft/models.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Resources
+---------
+
+A given table for ``resourceX`` (e.g., ``node``, ``DB``, ``FTP``) will have the
+following fields of the given type:
+
+=========================== =========== ======= =================================
+Field                       Type        Unique  Description
+--------------------------- ----------- ------- ---------------------------------
+resourceX_id                integer     true    Resources unique ID.
```
Comment (Kennric) [inline docs/source/draft/models.rst]: This field is just 'id', it doesn't need the resource name

File: docs/source/draft/models.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Measurements
+------------
+
+A given table for ``measurementX`` (e.g., ``CPU``, ``Memory``, ``Bandwidth``)
+will have the following fields of the given type:
+
+=============== ============ ======= ===================================================
+Field           Type         Unique  Description
+--------------- ------------ ------- ---------------------------------------------------
+measurementX_id integer      true    Measurement unique id.
```
Comment (Kennric) [inline docs/source/draft/models.rst]: Again, this is really just the standard id for this table, no need for measurement_id

File: docs/source/draft/models.rst [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Plugins
+-------
+
+Plugins are registered in the database with the following schema:
+
+================== ======= ====== =================================================================
+Field              Type    Unique Description
+------------------ ------- ------ -----------------------------------------------------------------
+plugin_id          integer true   Plugin unique id.
```
Comment (Kennric) [inline docs/source/draft/models.rst]: id

File: docs/source/draft/models.rst [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+plugin_name        string  true   Plugin unique name.
```
Comment (Kennric) [inline docs/source/draft/models.rst]: name

**Comment by Kennric** [comment]
Aside from minor issues noted above, I think this is a good first draft doc.

**Comment by matthewrsj** [comment]
These are good draft docs. +1.

**Pull request merged**